anaconda.py

```diff
@@ -405,6 +405,11 @@ def _earlyExceptionHandler(ty, value, traceback):
     if ksdata.rescue.rescue:
         flags.rescue_mode = True
 
+    # Check if we are running in rescue mode to avoid running some post
+    # scripts later.
+    if flags.rescue_mode:
+        util.touch('/tmp/RESCUE_MODE')
```

Comment:
Please move path to the file as variable to a `pyanaconda/core/constants.py`.

Reply:
Done